server/sonar-server/src/main/java/org/sonar/server/rule/ws/RuleWsSupport.java

```diff
@@ -44,7 +44,7 @@ public RuleWsSupport(DbClient dbClient, UserSession userSession, DefaultOrganiza
     this.defaultOrganizationProvider = defaultOrganizationProvider;
   }
 
-  public void checkQProfileAdminPermission() {
+  public void checkQProfileAdminPermissionOnDefaultOrganization() {
```

Comment:
If I understand correctly, this renaming is not fixing the problem, it's just here to make things clearer
I agree on the change but it should be a dedicated commit